### ISSUE TITLE
move command?

### ISSUE THREAD

**Issue by lukdiekm** [Regul777/linak-desk-cli#1]
Hi,

is there a move-command to move the desk to a new position?

Thanks

**Comment by Regul777** [comment]
No, not in the repository. I have this code and can push it as-is. It's a garbage, but works :)

**Comment by lukdiekm** [comment]
That would be great!
